### DISCUSSION TITLE
How to Sort results with Created date when Xrank is also there

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4140] [General]
I am using Xrank in below search query but want to sort by created also, I tried below query but its not returning any result

{searchTerms} XRANK(cb=500) DepartmentId:"xxxxxxxxxxxxxxxxxxxxxxxxx" (Path:"https://domain1.sharepoint.com/" OR Path:"https://doamin2.sharepoint.com/" OR Path:"https://domain1-my.sharepoint.com/") orderby=Created desc

**Comment by kasperbolarsen** [comment]
As far as I know you can choose to get the results either by Rank OR by Modified, not both

**Reply by wobba** [reply]
Use nb and not cb as XRANK parameter. And you should add some () to ensure your boolean logic works as expected and remove the "orderby" part which is not valid KQL. And you need to sort on rank for XRANK to work.
